docs/package.json

```diff
@@ -18,8 +18,8 @@
     "clsx": "^1.1.1",
     "docusaurus-gtm-plugin": "^0.0.2",
     "prism-react-renderer": "^1.3.3",
-    "react-dom": "^17.0.2",
     "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```

Comment:
automatic resorting of deps, no actual change in /docs

Reply:
I tried to replicate this re-sorting behavior, but I see `*.json` is included in the `.prettierignore` file. How did these re-sort automatically?

Reply:
they actually re-sorted when I was `npm installing` stuff, not from prettier. <shrug>